### PR TITLE
Add diagnostics documentation

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Adds a basic page for the new diagnostics integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/diagnostics.markdown [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+<img class="no-shadow" src='/images/blog/2022-02/diagnostics.png' alt='Screenshot showing the Download Diagnostics button on a Sonos device page'>
+Screenshot of the Download Diagnostics button on the device page
+</p>
+
+Diagnostics data is provided by an integration, and can be downloaded
+as a text file on the device page, and from the integrations dashboard.
+
+Integrations are required to redact sensitive information from the diagnostics
+data, you can always open the text file and check it, before you send
+it in an public issue.
```
Comment (ludeeus) [inline source/_integrations/diagnostics.markdown]:
```suggestion
it in a public issue.
```

**Review by ludeeus — DISMISSED** [review]

File: source/_integrations/diagnostics.markdown [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Diagnostics
+description: The diagnostics integration can provide integration and device information for debugging purposes.
+ha_category:
+  - Other
```
Comment (raman325) [inline source/_integrations/diagnostics.markdown:5]: Remote Python Debugger is in the Utility category. I know these aren't the same things but they seem to be in a similar vein of troubleshooting no? Should they be in the same category then?

Comment (frenck) [reply source/_integrations/diagnostics.markdown:5]: Dunno, this is now in the same category as System Health.

Comment (ludeeus) [reply source/_integrations/diagnostics.markdown:5]: Being in the same as system health make sense 👍

**Review by ludeeus — APPROVED** [review]

**Pull request merged**